### PR TITLE
Proof updates and adding DP chapter

### PR DESCRIPTION
**Pull request by jimhshen**

### CODE REVIEW

File: 01_intro.Rmd [hunk 1]
```diff
@@ -178,41 +178,41 @@ if ( knitr::is_latex_output() ) {
...
+Many **mature systems for research data access are hosted by universities that collaborate with specific governments**. Aside from the Stanford-SFUSD Partnership and the IAB, another example of this in the Handbook is the OLDA. The advantages of hosting the data at academic institutions are many: they often have an ethics review board (IRB) or can provide support for ethics review, they manage grants, they can supply space and an existing computing infrastructure, and can provide channels to other researchers as well as audiences (conferences, seminars, plenary discussions, events, etc.). Postdoctoral researchers and graduate and undergraduate students can contribute their skills to the data work; access to the data for their own research may provide additional incentives. Universities are often seen as more independent and less political or partisan than other policy research organizations such as think tanks. Chapter \@ref(olda) describes how OLDA's institutionalization as a center at Ohio State University facilitated long-term research projects across legislative cycles and associated changes in policy priorities.
```
Comment (larsvilhuber) [inline 01_intro.Rmd:185]: ... two other examples of this in the Handbook are the OLDA and the NB-IRDT.  .... changes in policy priorities. Chapter \@ref(nbirdt) is a newer example of a similar arrangement in New Brunswick province in Canada.

File: 02_security.Rmd [hunk 2]
```diff
@@ -292,11 +292,11 @@ The terms of the remote access will be defined in the DUA between the researcher
...
 
-When analysis methods are **unrestricted**, researchers can use the full set of methods available in the software that are provided on the analysis computer, including any tabulation or regression analysis. Note that the ability to report on the results obtained via these methods might still be restricted, depending on what is considered safe output. Furthermore, the ability to access any method, for instance through add-on packages (e.g., SSC for Stata or CRAN for R) may depend on the agency the researcher has over the analysis computer.
+When analysis methods are **unrestricted**, researchers can use the full set of methods available in the software that are provided on the analysis computer, including any tabulation or regression analysis. Note that the ability to report on the results obtained via these methods might still be restricted, depending on what is considered safe output. Furthermore, the ability to access any method, for instance through add-on packages such as Statistical Software Components (SSC) for Stata or Comprehensive R Archive Network (CRAN) for R, may depend on the agency the researcher has over the analysis computer.
```
Comment (larsvilhuber) [inline 02_security.Rmd:299]: ... add-on packages distributed through repositories such as the Statistical Software Components (SSC) archive at Boston College for Stata or the Comprehensive R Archive Network (CRAN) for R, ...

File: assets/security/mechanisms.csv [hunk 3]
```diff
@@ -1,15 +1,15 @@
 name,agency,location,access,sec,range
-NB-IRDT,Medium,Third-Party,Data Custodian,High Security,Limited
-IAB RDC,Medium,Data Provider,Third-Party,High Security,Limited
-IAB JoSuA,Medium,Third-Party,Researcher,Low Security,Limited
-IAB SUF,High,Researcher,Researcher,Medium Security,Unrestricted
-OLDA,High,Researcher,Researcher,Low Security,Unrestricted
-PCRI,Medium,Data Provider,Researcher,Low Security,Limited
+NB-IRDT (chapter \@ref(nbirdt)),Medium,Third-Party,Data Custodian,High Security,Limited
```
Comment (larsvilhuber) [inline assets/security/mechanisms.csv:2]: Isn't a <br> needed in all the first columns?

**Review by larsvilhuber — APPROVED** [review]
@jimhshen  See a few minor changes. I'll merge it in, but they should be addressed.

File: assets/security/mechanisms.csv (hunk 3 above)
Comment (jimhshen) [reply assets/security/mechanisms.csv:2]: Isn't a what needed?

**Pull request merged**